error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/StringRulesTestOutput.java

```diff
@@ -39,16 +41,20 @@ ImmutableSet<Boolean> testStringIsNullOrEmpty() {
         Strings.isNullOrEmpty(getClass().getName()), !Strings.isNullOrEmpty(getClass().getName()));
   }
 
+  // XXX: Consider updating qualifier `Predicate#not` if/when conditional `ImportPolicy` is
+  // introduced.
   ImmutableSet<Optional<String>> testOptionalNonEmptyString() {
     return ImmutableSet.of(
-        Optional.ofNullable(toString()).filter(s -> !s.isEmpty()),
-        Optional.ofNullable(toString()).filter(s -> !s.isEmpty()),
-        Optional.ofNullable(toString()).filter(s -> !s.isEmpty()),
-        Optional.ofNullable(toString()).filter(s -> !s.isEmpty()));
+        Optional.ofNullable(toString()).filter(Predicate.not(String::isEmpty)),
+        Optional.ofNullable(toString()).filter(Predicate.not(String::isEmpty)),
```

Comment:
Should be a candidate for static import later.
See [here](https://github.com/PicnicSupermarket/error-prone-support/blob/ae22e0ec5e48dba12d93627058c2fd71150c9ff0/error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/StaticImport.java#L132)